### ISSUE TITLE
animes/hentai

### ISSUE THREAD

**Issue by ZaryImortal** [LaksCastro/anime-dart#2]
pensa em futuramente acrescentar alguma source de hentais? até hoje não encontrei nenhum app que suporte os dois.

**Comment by LaksCastro** [comment]
Sim, e já tentei encontrar uma API, mas a maioria dos sites de hentai não usa tecnologia de SPA, ou seja, fica bem difícil extrair somente os dados (sem crawler/webscraping) em json pois os dados vem embutidos no HTML da página.

Mas aí entra uma parte bacana, se você conhece algum site que carrega o conteúdo sem precisar de reload na página, me envie a URL, assim posso estudar a API do site para implementar no aplicativo e disponibilizar pra download.

Nota: não precisa ser exatamente uma SPA, basta que a API retorne Json (ou até XML).

Obrigado pelo interesse, vou pesquisar mais sobre!

**Comment by ZaryImortal** [comment]
Pra não criar outra issue vou perguntar aqui, pensa em colocar integração com o MyAnimeList?

**Comment by LaksCastro** [comment]
Não tinha pensado nisso, mas é uma ótima ideia. Ainda preciso ver como faria para integrar essas API's e fazerem funcionar juntas. Mas gostei bastante da ideia, isso acaba com a necessidade de usar o próprio celular para armazenar os favoritos/assistidos, vou pensar em uma forma de implementar.

**Comment by ZaryImortal** [comment]
tem um app de manga que eu uso que tem integração, também é código aberto, vc poderia dar uma olhada. https://github.com/inorichi/tachiyomi/

**Comment by LaksCastro** [comment]
Um repositório deveras interessante amigo(a). Obrigado por compartilhar, não conhecia este repositório tachiyomi. Tive algumas ideias que vou implementar nesta próxima release que vão ser bem bacanas, incluindo as integrações que citamos anteriormente. Pode enviar mais conteúdos e features que você pode desejar, como esse aplicativo é algo também utilizo pessoalmente, qualquer ideia de melhoria, por pequena que seja é totalmente bem vinda! =D
